docs/experiments-microblog.md

```diff
@@ -0,0 +1,44 @@
+# Anserini Experiments on TREC Microblog Collection
+
+
+### Tweets2011 collection
+
+Note that the Tweets2011 collection is distributed as a list of tweet ids that you have to download yourself, so the 
+effectiveness results you'll get should be similar, but will likely not be identical to the scores reported here.
+
+Indexing the Tweets2011 collection:
+
+```
+nohup sh target/appassembler/bin/IndexCollection -collection TwitterCollection  -input \
+/path/to/Tweets2011/ -generator JsoupGenerator  -index lucene-index.Tweets2011.pos+docvectors -threads 32 \
+-storePositions -storeDocvectors -optimize > log.Tweets2011.txt &
+```
```

Comment:
I'm in two minds, yes protecting against copy+paste is a good thing, but then that could also lead to surprises. For example if I copy+paste this, then it gets backgrounded and I get a new prompt, there's no output because it's all redirected (`nohup` redirects regardless of whether you add the redirection yourself). I _could_ then think that nothing happened and that it's broken.

I'm also against protecting against copy+paste because it's bad practice as a developer to blindly do it without looking at what's being performed.

The simplest way to address this is to just say that it's backgrounded and that you can come back to it. Of course if it's consistent with all the other Anserini docs, then perhaps best leave it as a personal annoyance more than an issue that needs addressing, after all consistency is perhaps the most important thing.